### ISSUE TITLE
Error when using plugin

### ISSUE THREAD

**Issue by rhigdon** [brendon1555/setup-linode-cli#13]
Trying to use this plugin and I get the following error:

![image](https://user-images.githubusercontent.com/133530/127685110-29d3559c-b568-4d99-8a0f-e6ad22d7a387.png)

```
name: test
# Controls when the workflow will run
on:
  # Triggers the workflow on push or pull request events but only for the main branch
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  linode:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@main
    - name: Setup Linode cli
      uses: brendon1555/setup-linode-cli@v1.0.2
      with:
        LINODE_CLI_TOKEN: ${{ secrets.LINODE_CLI_TOKEN }}
    - run: linode-cli --help
```

I believe the secrets file is correct.  Any idea what is causing these errors?

**Comment by brendon1555** [comment]
This is an issue from my end. I'll take a look at this shortly.
I need to change the way the token is being read in the action. The current way has been deprecated.

**Comment by brendon1555** [comment]
This should now be fixed in `v1.0.3`
